### ISSUE TITLE
Don't require a vendoring "solution" if there's a vendor folder

### ISSUE THREAD

**Issue by chendrix** [cloudfoundry/go-buildpack#37]
**What version of Cloud Foundry are you using?**

PWS

**What version of the buildpack you are using?**

1.7.5

**If you were attempting to accomplish a task, what was it you were attempting to do?**

Push the [`cli-plugin-repo`](https://github.com/cloudfoundry-incubator/cli-plugin-repo)

**What did you expect to happen?**

That it wouldn't require Godeps, since it's got a vendor folder with all of its dependencies met AND the manifest.yml file used specifies `GO15VENDOREXPERIMENT`. I was working off information found in #25 

**What was the actual behavior?**

```
Downloaded go_buildpack
Successfully created container
Downloading app package...
Downloaded app package (3.2M)
Staging...
-------> Buildpack version 1.7.5
file:///tmp/buildpacks/d70cf8c00babea7058f4ebb7870f57af/dependencies/https___pivotal-buildpacks.s3.amazonaws.com_concourse-binaries_godep_godep-v62-linux-x64.tgz
 !     https://github.com/cloudfoundry/go-buildpack
Failed to compile droplet
 !     Godep is required. For instructions:
Exit status 223
Staging failed: Exited with status 223

FAILED
BuildpackCompileFailed
```

Please confirm where necessary:
* [X] I have included a log output 
* [X] My log includes an error message 
* [X] I have included steps for reproduction

**Comment by jvshahid** [comment]
Thanks @chendrix for reporting this bug. I verified that this is indeed a bug.

**Comment by jvshahid** [comment]
@chendrix this bug has been fixed on develop and will be available in `1.7.7`. **Note** we released version `1.7.6` with the fix, but turns out there is another bug introduced in that version. You should wait for `1.7.7` to be released.
